server/app/services/program/ProgramService.java

```diff
@@ -278,6 +278,21 @@ public ProgramDefinition getDraftFullProgramDefinition(String programSlug)
     return syncProgramAssociations(draftProgram).toCompletableFuture().join();
   }
 
+  /**
+   * Get the definition of a given program. Gets the active or draft version for the slug (looks for
+   * active first; gets draft if active doesn't exist).
```

Comment:
question (maybe blocking): i think this comment is backwards. based on the code it looks like it returns the draft version first, then the active version. i tested it out manually and it seemed to return the draft version first. what's the intended behavior?